scripts/license-configuration.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
```

Comment:
script is not passing on https://www.shellcheck.net/

Reply:
Fixed.

>No issues detected!